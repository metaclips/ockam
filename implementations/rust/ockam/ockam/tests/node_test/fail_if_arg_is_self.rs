#[ockam::test]
async fn my_test(self) -> ockam_core::Result<()> {}

fn main() {}
