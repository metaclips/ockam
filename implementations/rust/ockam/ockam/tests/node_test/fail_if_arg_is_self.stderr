error: the function has no `Context` argument
 --> tests/node_test/fail_if_arg_is_self.rs:2:18
  |
2 | async fn my_test(self) -> ockam_core::Result<()> {}
  |                  ^^^^
